src/app/admin/ownership/ownership.component.html

```diff
@@ -10,7 +11,7 @@
         </div>
         <div class="row">
             <div class="col-md-8">
-                <input class="form-control w-100 p-4" id="platform-manager-address"
+                <input class="input-reg w-100" id="platform-manager-address"
```

Comment:
how many different input styles do we have inside the application? is input-reg different from input-lg? just asking if we can use the same classes inside any component so we have consistent inputs.